clustered/client/src/main/java/org/ehcache/clustered/client/config/ClusteringServiceConfiguration.java

```diff
@@ -40,6 +41,8 @@
     implements ServiceCreationConfiguration<ClusteringService>,
     CacheManagerConfiguration<PersistentCacheManager> {
 
+  private static final String CLUSTER_URI_PATTERN = "terracotta://([^\\]\\[/?#@]+@)?[^:?#/]+(:[1-9][0-9]{0,4})?(/[^\\?#]*)?(\\?[^#]*)?(#.*)?";
```

Comment:
`terracotta:` is used all over the place, let's not change that so late - we may open up to a shorter version later.